.github/workflows/docsbuild-release.yaml

```diff
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
+        description: 'Release Version. Example : 11.1.0'
+        required: true
+      ansible_major_version:
+        description: 'Exmaple 11'
+        required: true
+
+env:
+  CI_COMMIT_MESSAGE: 'Add Ansible community "${{ inputs.ansible_version }}" porting guide'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
First of all, I'm pretty sure `${{ env.GITHUB_TOKEN }}` is already present in the environment variables. So this shouldn't be necessary. But if it is, you must scope this to one step or even command. By setting this here, you make the token globally readable by all the processes in the job, making it more likely to leak.

Reply:
So it's not present in the env by default, but the point still stands — the secret exposure must be tightly scoped.

Reply:
Updated.

Reply:
Yes, I messed up in my local branch pushed again.